_posts/en/2022-02-03-project-to-product.md

```diff
@@ -0,0 +1,250 @@
+<!-- cSpell:ignoreRegExp /\<[^\>]+\>/ -->
+<!-- The img + em {} stylecheet selector is a hack to add caption to an image in markdown without using plugin: https://stackoverflow.com/questions/19331362/using-an-image-caption-in-markdown-jekyll -->
+
+<style>
+table, th, td {
+  border: 1px solid black;
+}
+
+th {
+  background-color: #ccccff;
+}
+
+img + em { display: inline-block; }
+
+img.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+</style>
+
+In this blog, we [continue to extrapolate]({{ site.baseurl }}/2021/10/15/product-management-part-2.html) on how product management can be implemented within the federal government, particularly in the digital space.
+
+<!--more-->
+
+We will follow how [Job Bank](https://www.jobbank.gc.ca/home)[^1] is transitioning from project to product by being provided with the necessary environmental structure to work in.
+We can tie this back to its [program desired outcomes](https://dialogue/grp/WG6278300/Program%20Inventory/Forms/AllItems.aspx?RootFolder=/grp/WG6278300/Program%20Inventory/Job%20Bank&FolderCTID=0x012000B9ED43B8F503194E9CF21B1ED35BB73E&View=%7b12E60D67-CC97-44CE-BBCA-B6E4EE0365E5%7d).
+
+_Note: This blog will link to internal ESDC documents, which are unfortunately only accessible within ESDC corporate network._
+
+Employment and Social Development Canada (ESDC) is a [service department](https://www.tbs-sct.gc.ca/ems-sgd/edb-bdd/index-eng.html#infographic/dept/128/services), as are the large majority of Government of Canada departments.
+ESDC provides its services via [50 Programs](https://dialogue/grp/WG6278300/SitePages/Performance%20Information%20Profiles.aspx) to [improve the standard of living and quality of life for all Canadians](https://www.canada.ca/en/employment-social-development.html).
+One of those programs is Job Bank.
+
+This is how we tie product to service and then to programs.
+
+<figure>
+<a href="{{ site.baseurl }}/assets/images/product-service-mgt.png">
+        <img class="center" src="{{ site.baseurl }}/assets/images/product-service-mgt.png"
+        alt="This image depicts a the relationship between product management and service management. The image shows two rectangles: one labeled 'Product mgt' with the text Managing internal complexities to upkeep product(s) necessary for service delivery; and the other labeled 'Service mgt' with the text Facilitates outcomes that clients want to achieve, without them having to manage specific costs and risks. There's an arrow pointing from the Product mgt rectangle to the Service mgt rectangle labled 'Enables' and with the text Services are based on one or more products. This image was altered from © αβ2ϲ and © AXELOS Limited 2019."
+        />
+  </a>
+    <figcaption>Figure 1. The Product - Service relationship</figcaption>
+</figure>
+
+A service consumer doesn't care about the costs and risks to operate the service, but a service uses products, which DO have a cost, for its delivery.

```

Comment:
Not sure if I caught this in the guidance doc but I'm wondering if the DO should be in lowercase and bold instead of caps?